frontend/store/companies/actions.js

```diff
@@ -0,0 +1,76 @@
+import types from "./mutation-types";
+
+export const fetchData = async (url) => {
+  const response = await fetch(url);
+  return response.json();
+};
+
```

Comment:
You added this one in `functions.js`, but still using that local one. Please, remove it and use wrapper from `functions.js`.

Reply:
Oh yeah, I think i went back with ctrl + z too much and skipped this problem. Will fix in a minute